src/block_cache/block_chunkcache.c

```diff
@@ -181,11 +181,11 @@ __chunkcache_bitmap_free(WT_SESSION_IMPL *session, size_t index)
 }
 
 /*
- * __chunkcache_should_pin_chunk --
+ * __name_in_pinned_list --
  *     Return true if the chunk belongs to the object in pinned object array.
  */
 static inline bool
-__chunkcache_should_pin_chunk(WT_SESSION_IMPL *session, WT_CHUNKCACHE_CHUNK *chunk)
+__name_in_pinned_list(WT_SESSION_IMPL *session, const char *name)
```

Comment:
There should be __chunkcache prepended (that seems to be the convention)

Reply:
I thought about this as well - but it seems like we actually have two other functions without the __chunkcache prefix so I am not sure if its convention? See __hash_id_eq and __config_get_sorted_pinned_objects. 

Let me know your thoughts.

Reply:
It's convention-ish. It doesn't matter for static functions since they won't be visible outside the chunk cache. I'm OK with it as-is, in fact there's probably a bit too much stuff with the `chunkcache_` prefix already in the code.

Reply:
ahh, I see, in that case, that's okay.